nanodbc/nanodbc.cpp

```diff
@@ -259,17 +262,25 @@ inline void convert(const wide_string& in, std::string& out)
 #ifdef NANODBC_ENABLE_BOOST
     using boost::locale::conv::utf_to_utf;
     out = utf_to_utf<char>(in.c_str(), in.c_str() + in.size());
-#else
-// Workaround for confirmed bug in VS2015 and VS2017 too
-// See: https://connect.microsoft.com/VisualStudio/Feedback/Details/1403302
-#if defined(_MSC_VER) && (_MSC_VER >= 1900)
+#elif defined(NANODBC_USE_STDLIB)
+    std::vector<wchar_t> characters(in.length());
+    for (size_t i=0; i<in.length(); i++)
```

Comment:
* Code formatting is inconsistent with `.clang-format`
* What's the reason behind hand-rolling the loop?